frontend/public/components/utils/pdb.tsx

```diff
@@ -0,0 +1,8 @@
+import { LabelSelector } from '@console/internal/module/k8s';
+
+export const getPDBResource = (pdbResources, resource) =>
+  pdbResources.find((f) =>
+    new LabelSelector(f.spec.selector).matchesLabels(
+      resource?.spec?.template?.metadata?.labels || resource?.metadata?.labels || {},
```

Comment:
I'm a little hesitant to always fall back to `metadata.labels` regardless of the resource. What workload kind do we need to check `resource.metadata.labels`?